### PR TITLE
Compute matrices from paper

### PR DESCRIPTION
**Pull request by jokasimr**
This PR adds a notebook for computing the matrices in the worflow described in [the paper](https://www.epj-conferences.org/articles/epjconf/abs/2023/12/epjconf_ecns2023_03004/epjconf_ecns2023_03004.html).

### CODE REVIEW

File: docs/examples/inverse_of_polarization_matrices.ipynb
Comment (SimonHeybrock) [inline docs/examples/inverse_of_polarization_matrices.ipynb:1]: Can you add also `(T_A * T_B).inv()`, which is needed for the case with 2 helium cells? @astellhorn should review the rest for correctness.

File: docs/examples/inverse_of_polarization_matrices.ipynb [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e5339fe5-2b2f-48a6-aa1e-2f30b319cc38",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "(TP.inv() @ F1.inv() @ F2.inv() @ TA.inv()).simplify()"
```
Comment (astellhorn) [inline docs/examples/inverse_of_polarization_matrices.ipynb:136]: As commented above: only needed in other beamline cases, at DREAM (the present workflow) we do not have flippers F1 and F2

File: docs/examples/inverse_of_polarization_matrices.ipynb [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5f85c7c9-e31e-420d-bf18-b2c9b7c0879c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "TA.inv()"
+   ]
+  },
```
Comment (astellhorn) [inline docs/examples/inverse_of_polarization_matrices.ipynb:75]: Similar to the code with flippers below, here adding "(TP.inv() @ TA.inv()).simplify()"? Or as Simon has proposed: ((Ta*Tp).inv()).simplify()?

**Review by astellhorn — APPROVED** [review]
All correct, only that we do not need the flippers for this current workflow, we do not have f1 and f2 here. But they will be needed in future for other beamlines

**Pull request merged**